CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+## Bug fixes
+
+Issues can be found on our [GitHub issues](https://github.com/yummygum/react-flagpack/issues) page. Found a bug? Let us know! Report the bug by [creating a new issue](https://github.com/Yummygum/react-flagpack/issues/new/choose). Make sure to check the issues page before creating a new bug issue to prevent duplicate issues. Once you start fixing a bug, post your intent on the issue to prevent others diving in at the same time.
```

Comment:
Do we have a system system in place regarding issue tags? For instance `bug` or `feature-request`. Whether we're using the standard Github tags or using our own, we should mention and explain how people can use these tags.

Reply:
There are two different types of issues that people can make (Bug report and Feature request). Also the labels will explain which one they need to use. I think it's very self-explaining.

![image](https://user-images.githubusercontent.com/10121174/104707157-57487800-571c-11eb-8418-fb7ffb7cfbc7.png)